doc/nrf/glossary.rst

```diff
@@ -0,0 +1,426 @@
+   It includes a public REST API that can be used for building IoT solutions.
+   See `nRF Connect for Cloud`_.
+
+nRF repository
+   An |NCS| repository that does not have an externally maintained, open-source upstream.
+   It is exclusive to Nordic development.
+
+nRF Secure Immutable Bootloader (NSIB)
+   A bootloader created and maintained by Nordic Semiconductor that is built on Chain of Trust architecture.
+
+OpenAMP
+   A framework that provides software components that enable the development of software applications for Asymmetric Multiprocessing (AMP) systems.
+   See `OpenAMP`_.
+
+OpenThread
+   A portable and flexible :ref:`open-source implementation <OpenThread.io>` of the Thread networking protocol.
+
+OpenThread Border Router (OTBR)
+   A router that connects a Thread network to other IP-based networks, like Wi-Fi or Ethernet.
+   A Thread network requires a Border Router to connect to other networks.
+
+OSS repository
+   An |NCS| repository that tracks an upstream Open Source Software counterpart that is externally maintained.
+
+Over-the-Air (OTA)
+   Any type of wireless transmission.
+
+Patch
+   See Commit.
+
+Power Amplifier (PA)
+   A device used to increase the transmit power level of a radio signal.
+
+Power Saving Mode (PSM)
+   A feature introduced in 3GPP Release 12 to improve battery life of IoT (Internet of Things) devices by minimizing energy consumption.
+   The device stays dormant during the PSM window.
+
+Printed Circuit Board (PCB)
+   A board that connects electronic components.
+
+Protocol Data Unit (PDU)
+   Information transferred as a single unit between peer entities of a computer network and containing control and address information or data.
+   PDU mode is one of the two ways of sending and receiving SMS messages.
+
+Provisioning
+   For a mesh network, this is the process of adding devices to the network.
+   For bootloaders, this is the process of storing public key hashes in a separate region of memory from the bootloader image.
+
+Pull Request
+   A GitHub Pull Request, a set of commits that are sent for code review using GitHub.
+
+Quality of Service (QoS)
+   The measured overall performance of a service, such as a telephony or computer network, or a cloud computing service.
+
+Radio Co-Processor (RCP)
+   A co-processor offloading radio functions from the host processor.
+   In the |NCS| context, it is typically used in OpenThread and Zigbee platform designs.
+
+Real-time operating system (RTOS)
+   A computing environment that reacts to input within a specific period of time.
+   A real-time deadline can be so small that system reaction appears instantaneous.
+
+Real-Time Transfer (RTT)
+   Proprietary technology for bidirectional communication that supports J-Link devices and ARM-based microcontrollers, developed by SEGGER Microcontroller.
```

Comment:
```suggestion
   Proprietary technology for bidirectional communication that supports J-Link devices and Arm-based microcontrollers, developed by SEGGER Microcontroller.
```
This is how the company Arm spells its name nowadays.

Reply:
fixed.